.eslintrc

```diff
@@ -45,6 +45,31 @@
 			{
 				"required": "id"
 			}
-		]
+		],
+		/* partially disable rules to get @woocommerce/eslint-plugin integration done */
+		"jsdoc/check-tag-names":                   "off",
+		"jsdoc/no-undefined-types":                "off",
+		"jsdoc/require-param":                     "off",
+		"jsdoc/check-param-names":                 "off",
+		"jsdoc/require-property":                  "off",
+		"@wordpress/no-unused-vars-before-return": "off",
+		"@wordpress/i18n-translator-comments":     "off",
+		"@wordpress/valid-sprintf":                "off",
+		"@woocommerce/dependency-group":           "off",
+		"react-hooks/exhaustive-deps":             "off",
+		"react-hooks/rules-of-hooks":              "off",
+		"react/jsx-key":                           "off",
+		"jest/no-conditional-expect":              "off",
+		"jest/valid-title":                        "off",
+		"jest/expect-expect":                      "off",
+		"jest/no-disabled-tests":                  "off",
+		"jest/no-standalone-expect":               "off",
+		"jest/no-identical-title":                 "off",
+		"jest/no-deprecated-functions":            "off",
+		"prettier/prettier":                       "off",
+		"yoda":                                    "off",
+		"no-alert":                                "off",
+		"object-shorthand":                        "off",
+		"no-multi-str":                            "off"
```

Comment:
Are those are the rules from the added packages we're breaking at the moment and need to be fixed in the src code?

I'd say it's fine to disable only completely new rules but if there are rules which existed before and fail now because config/severity changed we should manually set previous config rather than disabling them at all.

Reply:
Not only. My understanding is that the new package brings configuration changes for already available rules as well.
Ok, I'll re-visit the list of deactivated rules and restore the original configuration.

Reply:
I reactivated rules which seem to be available before + fixed violations for them. The remaining deactivations are clearly new rules we are getting with this PR.